scripts/swadeNpcImporter.js

```diff
@@ -29,11 +29,8 @@ Hooks.on('ready', async () => {
 
 Hooks.on('renderActorDirectory', async (app, html, data) => {
   if (game.users.get(game.userId).can('ACTOR_CREATE') == true) {
-    const npcImporterButton = $(
-      `<button style="margin: 4px; padding: 1px 6px;"><i class="fas fa-file-import"> ${game.i18n.localize(
-        'npcImporter.HTML.ActorImporter'
-      )}</i></button>`
-    );
+    const npcImporterButton = `<button style="width: calc(100% - 8px);"><i class="fas fa-align-left"></i>${game.i18n.localize('npcImporter.HTML.StatBlockImporterTitle')}</button>`;
+
     html.find('.directory-footer').append(npcImporterButton);
 
     npcImporterButton.click(() => {
```

Comment:
if you are already here, maybe update the `click` which is apparently deprecated, to :

`npcImporterButton.on('click', () => {`

Reply:
Done.